website/docs/d/autogen_subdomain.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "ns"
+page_title: "Nullstone: ns_autogen_subdomain"
+sidebar_current: "docs-ns-autogen-subdomain"
+description: |-
+  Data source to read a nullstone autogenerated subdomain.
+---
+
+# ns_autogen_subdomain
+
+Nullstone can generate autogen subdomains for users that look like `random-subdomain.nullstone.app`.
+This data source allows users to read information about that subdomain.
+
+## Example Usage
+
+#### Example
+
+```hcl
+data "ns_autogen_subdomain" "subdomain" {
+  name = var.subdomain
+}
+
+output "subdomain_fqdn" {
+  value = data.ns_autogen_subdomain.fqdn
+}
+```
+
+## Argument Reference
+
+- `name` - (Required) Name of auto-generated subdomain that already exists in Nullstone system. This should not include `nullstone.app`.
+
+## Attributes Reference
+
+* `domain_name` - The name of the domain that Nullstone administers for this auto-generated subdomain.

```

Comment:
I don't see this in the example above, is this still an attribute?

Reply:
It is still an attribute.
I included for future proofing; somebody would have to strip fqdn in terraform to get this value which is pretty ugly.